crates/egui/src/widgets/plot/items/mod.rs

```diff
@@ -682,6 +686,22 @@ impl Text {
         self
     }
 
+    /// Set font style of the text. Default is `TextStyle::Small`.
+    pub fn font_style(mut self, font: TextStyle) -> Self {
+        self.font = font;
+        self
+    }
+
+    /// Set Font size of the text.
+    /// If you want to use a custom font size, you can use `font_style` instead.
+    /// This will override the font size of the `font_style`.
+    /// The font key name is defined with prefixed by "Plot_" the "name" of `FondID` spaced
+    /// by a underscore and the "size" to prevent redundant `text styles` in ui.
+    pub fn font_size(mut self, font_size: f32) -> Self {
+        self.font_size = Some(font_size);
+        self
+    }
```

Comment:
A more general approach is to be able to set `FontId`. Users can still use `TextStyle` to get a `FontId` via `TextStyle::resolve`

Reply:
Thank you for these clarifications !